### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kontonkitsune**
Changed the description to actually describe the repo

### CODE REVIEW

**Review by bbucklanduno — COMMENTED** [review]
Nice

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# exercise2
\ No newline at end of file
+# exercise2
+
+This is the repository where the code for the various exercises is stored.
```
Comment (bbucklanduno) [inline README.md:3]: This is true

Comment (bbucklanduno) [inline README.md:3]: Hello

**Review by kaufatim — COMMENTED** [review]
This is the repository.

**Pull request merged**